README.md

```diff
@@ -222,6 +222,18 @@ You can use the following rules on your Android projects (see [Proguard Exclusio
     
     }
     ```
+* **Require Algorithm.** When parsing you can specify that **only** a particular signing algorithm can be used, giving control of algorithm selection to the entity doing the parsing and not allowing the token to choose the algorithm. This provides *some* protection against the "alg":"none" and RS256 as HS256 [attacks](https://auth0.com/blog/a-look-at-the-latest-draft-for-jwt-bcp/).
```

Comment:
Please do not reference 3rd party URLs other than the JWT RFC's, our own documentation, or de-facto/vetted StackOverflow answers, as 3rd-party sites can be unreliable or even provide incorrect information (The company URL you reference has had their fair share of [security](https://auth0.com/docs/security/bulletins/cve-2018-6874) [vulnerability](https://auth0.com/docs/security/bulletins/cve-2018-6873) [problems](https://thehackernews.com/2018/04/auth0-authentication-bypass.html) and aren't a reliable source of security information IMO).

Additionally, JJWT has never been susceptible to the referenced attack if you use the API as intended (call `parseClaimsJws`).